src/ripple/protocol/impl/STTx.cpp

```diff
@@ -286,25 +295,32 @@ STTx::checkSingleSign () const
     if (isFieldPresent (sfSigners))
         return false;
 
-    bool ret = false;
+    bool validSig = false;
     try
     {
-        ECDSA const fullyCanonical = (getFlags() & tfFullyCanonicalSig)
-            ? ECDSA::strict
-            : ECDSA::not_strict;
-
-        RippleAddress n;
-        n.setAccountPublic (getFieldVL (sfSigningPubKey));
+        bool const fullyCanonical = (getFlags() & tfFullyCanonicalSig)
+            ? true
+            : false;
```

Comment:
If I turn on /W4, I get 1046 warnings from VS rebuilding all. This does not appear to be one of them. (I usually build with /W3, and it gives no warnings.)